src/controllers/ChatsController.ts

```diff
@@ -43,9 +71,10 @@ export class ChatsController {
   async deleteChat(id: number) {
     try {
       await this.api.deleteChatById(id);
-      store.set('currentChatId', null);
       store.set(`messanges.${id}`, []);
```

Comment:
скорее всего опечатка
```suggestion
      store.set(`messages.${id}`, []);
```